akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorSpec.scala

```diff
@@ -507,12 +508,15 @@ class EventSourcedBehaviorSpec extends ScalaTestWithActorTestKit(EventSourcedBeh
     }
 
     "fail after recovery timeout" in {
-      EventFilter.error(start = "Persistence failure when replaying snapshot", occurrences = 1).intercept {
+      EventFilter[JournalFailureException](
```

Comment:
Is the only change in behavior the type of the exception?

Reply:
yes, previously it returned `Behaviors.stopped` instead of throwing `JournalFailureException` and therefore the supervisor was not involved, e.g. the specified backoff supervisor was not used for that case.